### PR TITLE
use docker/pkg/term to correct colored texts on Windows & create a function to print figlet

### LINKED ISSUE #289: Colored texts in PowerShell native console
## Current Behaviour
In PowerShell native console, ANSI escape are used and the PowerShell does not understand. The printed logs are not much readable.

With WSL (Windows Subsystem for Linux), we can still use the CLI correctly, but it's preferable that we can use it with the same UX in the PowerShell native console.

PowerShell native
PowerShell native

PowerShell in WSL - ubuntu
PowerShell in WSL - ubuntu

## Possible Solution
Use docker/pkg/term

### PR DESCRIPTION
**Pull request by itscaro**
## Motivation and Context

- [x] I have raised an issue to propose this change (resolve #289)

## How Has This Been Tested?
PowerShell & Bash in WSL
image

Debian 9 via Bash in WSL
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: commands/version.go [hunk 1]
```diff
@@ -40,16 +41,20 @@ func runVersion(cmd *cobra.Command, args []string) {
...
 		fmt.Printf("Version: %s\n", version.BuildVersion())
 	}
 }
 
+func printFiglet() {
+	figletColoured := aec.BlueF.Apply(figletStr)
+	if runtime.GOOS == "windows" {
+		figletColoured = aec.GreenF.Apply(figletStr)
+	}
+	ansi.Printf(figletColoured)
```
Comment (alexellis) [inline commands/version.go]: Thanks for the change. I just don't understand why this stopped working.. We also have to test this on Linux and Mac before merging.

Comment (itscaro) [reply commands/version.go]: @alexellis CI breaks because I did not commit vendored files

File: commands/faas.go [hunk 2]
```diff
@@ -47,6 +48,9 @@ func resetForTest() {
 }
 
 func init() {
+	// Setup terminal std
+	term.StdStreams()
```
Comment (itscaro) [inline commands/faas.go]: This will do the job :-)

File: commands/version.go (hunk 1 above)
Comment (dungeonmaster18) [reply commands/version.go]: It works propely with linux @itscaro . Just if you can add the dependency it would be great.

**Comment by itscaro** [comment]
Test results

**Comment by johnmccabe** [comment]
@itscaro LGTM, colour rendering as expected in Powershell 5.1.16299.98 and cmd.exe on Win10.0.16299.98

**Comment by alexellis** [comment]
Merged - thank you for this fix 👍

**Pull request merged**